mesh/accountsettings/accountsettings.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS AccountSettings(
```

Comment:
nitpick: tbh I think this should be called `Settings` as I don't think prefacing with `Account` adds any clarity